cmd/argocd-util/commands/projects.go

```diff
@@ -192,3 +199,83 @@ func updateProjects(projIf appclient.AppProjectInterface, projectGlob string, ro
 	}
 	return nil
 }
+
+func NewGenProjectConfigCommand() *cobra.Command {
+	var (
+		opts         cmdutil.ProjectOpts
+		fileURL      string
+		clientConfig clientcmd.ClientConfig
+		outputFormat string
+	)
+	var command = &cobra.Command{
+		Use:   "proj-create APPNAME",
+		Short: "Generate declarative config for a project",
+		Run: func(c *cobra.Command, args []string) {
+			var proj v1alpha1.AppProject
+			fmt.Printf("EE: %d/%v\n", len(opts.SignatureKeys), opts.SignatureKeys)
+			if fileURL == "-" {
```

Comment:
I've noticed with repeat this snippet of code 5 times (search for `if fileURL == "-" {` ) . Can you please create move it into a function ?